DEVELOPER_GUIDE.md

```diff
@@ -1,20 +1,21 @@
 - [Developer Guide](#developer-guide)
-  - [Getting Started](#getting-started)
-    - [Fork and Clone opensearch-api-specification Repo](#fork-and-clone-opensearch-api-specification-repo)
-  - [Use an Editor](#use-an-editor)
-    - [Visual Studio Code](#visual-studio-code)
-    - [Build](#build)
-  - [Adding a new API definition](#adding-a-new-api-definition)
-    - [Naming Convention](#naming-convention)
-    - [File Structure](#file-structure)
-    - [Defining the API model](#defining-the-api-model)
-  - [Adding a test-case for API definition](#adding-a-test-case-for-api-definition)
-    - [File Structure](#file-structure-for-test-folder)
-    - [Defining test-case for API model](#defining-test-case-for-api-model)
-  - [Local testing](#local-testing)
-    - [Pre-requisites](#pre-requisite)
-    - [Testing model API](#testing-model-api)    
-  
+	- [Getting Started](#getting-started)
+		- [Fork and Clone opensearch-api-specification Repo](#fork-and-clone-opensearch-api-specification-repo)
+	- [Use an Editor](#use-an-editor)
+		- [Visual Studio Code](#visual-studio-code)
+		- [Build](#build)
+		- [Formatting](#formatting)
+	- [Adding a new API definition](#adding-a-new-api-definition)
+		- [Naming Convention](#naming-convention)
+		- [File Structure](#file-structure)
+		- [Defining the API model](#defining-the-api-model)
+	- [Adding a test-case for API definition](#adding-a-test-case-for-api-definition)
+		- [File Structure for Test-folder](#file-structure-for-test-folder)
+		- [Defining test-case for API model](#defining-test-case-for-api-model)
+	- [Local testing](#local-testing)
+		- [Pre-requisite](#pre-requisite)
+		- [Testing model API](#testing-model-api)
```

Comment:
Are those tabs? :)

Reply:
Yeah :) I indented all the files in this repo with tabs including Smithy models. Should we go for spaces instead of tabs?

Reply:
Changed it to indenting to spaces.